Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.cpp

```diff
@@ -50,7 +50,9 @@ void HybridPICModel::ReadParameters ()
     pp_hybrid.query("plasma_resistivity(rho,J)", m_eta_expression);
     utils::parser::queryWithParser(pp_hybrid, "n_floor", m_n_floor);
 
-    utils::parser::queryWithParser(pp_hybrid, "plasma_hyper_resistivity", m_eta_h);
+    // Trying to add hyper-resistivity expression -  mtobin 20240217
+    // utils::parser::queryWithParser(pp_hybrid, "plasma_hyper_resistivity", m_eta_h);
+    pp_hybrid.query("plasma_hyper_resistivity(rho,B)", m_eta_h_expression);
```

Comment:
Can you please move this line up to be just below the line for the resistivity (line 50). Since these are now following the same pattern it would make sense to group them closer together.